src/osyris/plot/wrappers.py

```diff
@@ -40,12 +42,13 @@ def pcolormesh(ax, x, y, z, **kwargs):
     Wrapper around Matplotlib's pcolormesh plot.
     """
     default_args = {
-        "shading": "nearest",
+        "origin": "lower",
+        "extent": [np.min(x),np.max(x),np.min(y),np.max(y)]
     }
     default_args.update(kwargs)
     if "cmap" not in kwargs:
         kwargs["cmap"] = config.parameters["cmap"]
-    return ax.pcolormesh(x, y, z, **default_args)
+    return ax.imshow(z,**default_args)
```

Comment:
I would like to keep `pcolormesh` because it allows me to do things like setting a log-scale on the x or y axes, which is more tricky with `imshow`.